src/Symfony/Component/Notifier/Bridge/Smsapi/SmsapiTransport.php

```diff
@@ -64,7 +64,7 @@ public function setTest(bool $test): static
 
     public function __toString(): string
     {
-        return sprintf('smsapi://%s?from=%s&fast=%d', $this->getEndpoint(), $this->from, (int) $this->fast);
+        return sprintf('smsapi://%s?from=%s&fast=%d&test=%d', $this->getEndpoint(), $this->from, (int) $this->fast, (int) $this->test);
```

Comment:
I think it'd be better to not add neither test nor fast when they're not set

Reply:
I agree

Reply:
Or do you mean show only if set to true?

Reply:
Makes sense, I can copy paste it, which will result in false for `fast` and `test` 👍 

Will rework the PR